src/utils/config.js

```diff
@@ -32,6 +32,7 @@ let config = {
     defaultModalScroll: null,
     defaultDatepickerMobileNative: true,
     defaultTimepickerMobileNative: true,
+    defaultTimepickerMobileModal: true,
```

Comment:
It's a double

Reply:
Hi @jtommy,

The constants are created parallel to the datepicker constants, same prefix and structure.